src/CopyrighFixer/CopyrighFixer/sign.cpp

```diff
@@ -37,4 +45,115 @@ const QString &Signature::getMessage() const {
     return _customMessage;
 }
 
+bool Signature::fromJson(const QString &pathToFile) {
+
+    QFileInfo signFile(pathToFile);
+    if (signFile.exists() && signFile.isFile()) {
+
+        QFile file(pathToFile);
+        if (!file.open(QIODevice::ReadOnly)) {
+            QuasarAppUtils::Params::log("Json file couldn't be opened/found");
+            return false;
+        }
+
+        QByteArray byteSignFile(file.readAll());
+        file.close();
+
+        QJsonDocument jsDoc(QJsonDocument::fromJson(byteSignFile));
```

Comment:
remove excess variable from stack.
```suggestion
        QJsonDocument jsDoc(QJsonDocument::fromJson(file.readAll()));
        file.close();
        
```